FluentStorage/Blobs/Files/DiskDirectoryBlobStorage.cs

```diff
@@ -79,11 +79,13 @@ internal class DiskDirectoryBlobStorage : IBlobStorage {
 
 				var blob = new Blob(relPath, kind);
 				blob.Size = fi.Length;
-				blob.LastModificationTime = fi.CreationTimeUtc;
+				// Converting the local time to a DateTimeOffset will save the offset of UTC.
+				blob.LastModificationTime = fi.LastWriteTime;
+				blob.CreatedTime = fi.CreationTime;
 				blob.TryAddProperties(
 				   "IsReadOnly", fi.IsReadOnly.ToString(),
-				   "LastAccessTimeUtc", fi.LastAccessTimeUtc.ToString(),
-				   "LastWriteTimeUtc", fi.LastWriteTimeUtc.ToString(),
+				   // Universal sortable ("u") is always the same regardless of culture.
+				   "LastAccessTime", fi.LastAccessTimeUtc.ToString("u"),
```

Comment:
can you explain what this does please?

Reply:
I removed LastWriteTimeUtc (see next comment), However LastAccessTimeUtc in the extra data should be saved using a universal sortable format, else it will change depend on the users culture info (for example d/M/y or M/d/Y) depending on their locale.